object/slicer/options.go

```diff
@@ -55,15 +58,29 @@ func (x *Options) SetCopiesNumber(copiesNumber uint32) {
 	x.copiesNumber = copiesNumber
 }
 
-// SetPayloadBuffer sets pre-allocated payloadBuffer to be used to object uploading.
-// For better performance payloadBuffer length should be MaxObjectSize from NeoFS.
+// SetPayloadBuffer sets pre-allocated payloadBuffer to be used to object
+// uploading. The payloadBuffer should have length at least
+// [Options.ObjectPayloadLimit] + 1. If [Options.SetPayloadSize] is called, its
```

Comment:
i am not sure i understand these restrictions. both `+1` and less buffer ignorance. why cant a user put a 2X object with an X buffer (imagine it just already has it)?

Reply:
gone